### ISSUE TITLE
Compiling Monster64 for stm32h743

### ISSUE THREAD

**Issue by dashiad** [Jaystonian/Cartridges#3]
Hello!
I'm trying to port your port of kung-fu flash, to the stm32h743vti , and i'm finding a few problems building the code. I'm 100% new to stm32 development, so maybe i'm missing obvious things.
For example, I'm getting quite a few errors from the stm32h7xx/hal.c file, in the sysclk_config function, where there is this block of code:
```
RCC->APB
 RCC->APB1ENR |= RCC_APB1ENR_PWREN;
```
Is that first line a typo? 
In any case, the field APB1ENR doesnt look to be defined for the stm32h7xx (in the stm32h753xx.h file in the repo, do exist APB1LENR and APB1HENR, while APB1ENR looks like it's defined for the stm32f405...
So, maybe i shouldnt be buiding that file at all?

**Comment by berrydejager** [comment]
So we bought a doorstopper?

**Comment by Jaystonian** [comment]
> So we bought a doorstopper?

No.  This was a very limited run (6 made) as a development cartridge, since it is much more powerful than the Kung Fu Flash, hoping somebody would help port the code or get some help from the KFF designer, who basically gave me the finger.  That was supposed to be the idea of opensource.  It is still a functional cartridge, it just does not have the KFF code ported to it.  KFF uses the F chip and this board uses the more powerful H chip.  There are significant differences between the F and H series chips and STM seems to have chosen to provide a lot more support to the F chips.  The intended market for these cards are developers needing a bridge to the hardware.
